### PR TITLE
Update docs on library release

### PR DESCRIPTION
**Pull request by smyrick**
### :pencil: Description
We should update the docs versions on every code release. We can then come back manually and delete excess version numbers, but the docs should always reflect the latest

### :link: Related Issues
Original issue here: https://github.com/ExpediaGroup/graphql-kotlin/pull/767

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -34,3 +35,39 @@ jobs:
...
+          restore-keys: |
+            ${{ runner.os }}-node-
+
+      - name: Build website
+        run: |
+          NEW_VERSION=$(echo "${GITHUB_REF}" | cut -d "/" -f3)
+          echo "New version: ${NEW_VERSION}"
+          cd website
+          npm install
+          npm run version $NEW_VERSION
```
Comment (dariuszkuc) [inline .github/workflows/release.yml:64]: this actually creates new folder/files under `website` dir, if you want to auto publish those then I guess we need to explicitly add them to the repo? I believe the next action only pushes to `gh-pages`  branch.

Comment (smyrick) [reply .github/workflows/release.yml:64]: This action here does publish to the `gh-pages` branch

https://github.com/ExpediaGroup/graphql-kotlin/pull/852/files#diff-56c30d9224897b08c28075052d4595c5R68

Comment (dariuszkuc) [reply .github/workflows/release.yml:64]: yes action pushes to `gh-pages` but we need to push to master - update sidebars (https://github.com/ExpediaGroup/graphql-kotlin/tree/master/website/versioned_sidebars) and create versioned markdown files (https://github.com/ExpediaGroup/graphql-kotlin/tree/master/website/versioned_docs/version-3.6.1)

Comment (smyrick) [reply .github/workflows/release.yml:64]: Oh, right. I will update to use the following action: https://github.com/marketplace/actions/github-push?version=v0.6.0

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -34,3 +35,50 @@ jobs:
...
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          publish_dir: ./website/build/graphql-kotlin
+          user_name: eg-oss-ci
+          user_email: oss@expediagroup.com
+
+      - name: Commit new files
+        run: |
+          git config --local user.email "action@github.com"
+          git config --local user.name "GitHub Action"
+          git commit -a -m "Add new docs version"
```
Comment (dariuszkuc) [inline .github/workflows/release.yml:79]: instead of `-a` should we limit to some website sub directories (just in case)?

**Pull request merged**